tests/test_diffeq/test_perturbedsolvers/test_perturbation_functions.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+from probnum.diffeq.perturbedsolvers import _perturbation_functions
+
+random_state = np.random.mtrand.RandomState(seed=1)
+
+
+def test_perturb_uniform():
```

Comment:
it seems to me that the test checks whether the output is identical once the seed is fixed, right? I am not sure whether this is the best possible test here (it might actually be too close to the implementation of the test function, but this is only a hunch, not a clear statement)... 

Can we maybe also have some sort of test that checks the assumptions of the Abdulle/Garegnani paper (i.e. E(h_k) = h, Var())? That would make a good test across perturbation functions as well.

Reply:
Oh i see that you do this for the mean below. One for the variance would be good, too (because this is also an important part of the assumption!)